build-images.sh

```diff
@@ -169,4 +177,10 @@ if [ "$BUILD_SCHEDULER" = "YES" ]; then
     mkimage mama-ng-scheduler $SCHEDULER_DIR Dockerfile
 fi
 
+if [ "$BUILD_REGISTRATION" = "YES" ]; then
+    echo "Building registration image..."
+    mkimage hellomama-registration $REGISTRATION_DIR Dockerfile
```

Comment:
Just `mkimage hellomama-registration` should be fine.

The 2nd and 3rd argument should default to the correct values (the above values are incorrect).